packages/create-audius-app/package.json

```diff
@@ -47,6 +48,6 @@
     "tar": "6.2.0",
     "tsup": "8.0.2",
     "validate-npm-package-name": "5.0.0",
-    "vitest": "1.3.0"
+    "vitest": "0.34.6"
```

Comment:
just checking we want the downgrade?

Reply:
Yes! This caused me much frustration. Mismatched vitest version with the hoisted version of vite was causing a cryptic `Uknown node type: 677` error